src/metabase/models/card.clj

```diff
@@ -40,6 +40,13 @@
   [{:keys [id]}]
   (db/count 'DashboardCard, :card_id id))
 
+(defn filter-values-usage-count
```

Comment:
Looks like Cam's [PR](https://github.com/metabase/metabase/pull/27475) to migrate hydration functions to use multimethod is merged, so we should follow that and define this with `define-simple-hydration-method`